### ISSUE TITLE
Eliminate/remove the  ODE Accumulation Trigger to be replaced by new ODE Accumulator 

### ISSUE THREAD

**Issue by rjhowell44** [prominenceai/deepstream-services-library#749]
The ODE Accumulation Trigger accumulates the count of new object "Instances" and calls on its ODE Actions during the post-processing of each frame.  This functionality is required for more than just new instances metrics. Directional line crossing is an other prime example. 

The current ODE Accumulation Trigger functionality will be implemented  by adding an ODE Accumulator to an ODE Instance Trigger. 

depends on #748

**Comment by rjhowell44** [comment]
complete - merged into v0.24.alpha
